arch/arm/core/aarch64/reset.c

```diff
@@ -64,6 +88,14 @@ void z_arm64_el3_init(void)
 	z_arm64_el3_plat_init();
 
 	isb();
+
+	if (arm64_el3_get_next_el() == EL3_TO_EL1_SKIP_EL2) {
+		/* handle EL2 init in EL3,
+		 * as it still needs to be done
+		 * but we are going to be skipping EL2.
+		 */
```

Comment:
nit: I know that this is not mandate in the Zephyr coding style but can you change the multi-line comment to the linux kernel coding style?
```
/*
 * comment
 * comment
 */
```
and also fix the length of the lines.

Reply:
Done.